controllers/offer.js

```diff
@@ -2,7 +2,7 @@ const offerService = require('~/services/offer')
 const offerAggregateOptions = require('~/utils/offers/offerAggregateOptions')
 
 const getOffers = async (req, res) => {
-  const pipeline = offerAggregateOptions(req.query, req.params)
+  const pipeline = offerAggregateOptions(req.query, req.params, req.user)
```

Comment:
you can pass only req.user.id if you don't use any other data of it in aggregation function